packages/components/src/templates/header.stories.ts

```diff
@@ -57,41 +57,166 @@ export const SampleA = {
                 class="text-xl absolute -ml-[4px]"
               ></sd-icon>
             </div>
           </sd-navigation-item>
           <sd-navigation-item>
             <div class="w-4 h-4 flex items-center">
-              <sd-icon
-                title="Suche"
-                name="system/magnifying-glass"
-                library="global-resources"
-                class="text-xl absolute -ml-[4px]"
-              ></sd-icon>
+              <sd-icon title="Suche" name="system/magnifying-glass" class="text-xl absolute -ml-[4px]"></sd-icon>
             </div>
           </sd-navigation-item>
           <sd-navigation-item>
             <div class="flex items-center gap-2">
-              <sd-icon name="system/profile" library="global-resources" class="text-xl"></sd-icon
-              ><span>Mein Depot</span>
+              <sd-icon name="system/user" class="text-xl"></sd-icon><span>Portfolio</span>
+            </div>
+          </sd-navigation-item>
+          <sd-navigation-item>
+            <div class="flex items-center gap-2">
+              <sd-icon name="system/lock-locked" class="text-xl"></sd-icon><span>Application</span>
+            </div>
+          </sd-navigation-item>
+          <!-- bottom-right-area end !-->
+        </div>
+      </div>
+    </sd-header>
+    <sd-drawer id="sampleA__drawer" placement="end">
+      <nav>
+        <sd-navigation-item vertical current>
+          <b>Home</b>
+        </sd-navigation-item>
+        <sd-navigation-item vertical chevron divider>About Us</sd-navigation-item>
+        <sd-navigation-item vertical chevron divider>Markets</sd-navigation-item>
+        <sd-navigation-item vertical chevron divider>Sustainability</sd-navigation-item>
+        <sd-navigation-item vertical chevron divider>Career</sd-navigation-item>
+      </nav>
+      <nav slot="footer" class="bg-neutral-100">
+        <sd-navigation-item vertical class="flex align-center">
+          <sd-icon name="system/user" class="h-6 w-6 mr-2"></sd-icon>
+          Portfolio
+        </sd-navigation-item>
+        <sd-navigation-item vertical divider class="flex align-center">
+          <sd-icon name="system/lock-locked" class="h-6 w-6 mr-2"></sd-icon>
+          Application
+        </sd-navigation-item>
+        <sd-navigation-item vertical divider class="flex align-center">
+          <sd-icon name="system/website" class="h-6 w-6 mr-2"></sd-icon>
+          Our further appearances
+        </sd-navigation-item>
+      </nav>
+    </sd-drawer>
+    <script>
+      // This line is using querySelectorAll due to hidden elements from Storybook configs.
+      // In a regular environment, use: var drawer = document.getElementById('sampleA') or similar
+      var drawerSampleA = document.querySelectorAll('#sampleA__drawer')[1];
+      console.log(document.querySelectorAll('#sampleA__drawer')[1]);
+      function openDrawerSampleA() {
+        drawerSampleA.show();
+      }
+    </script>
```

Comment:
Make this a type=module and do not use `var` but const instead.

Remove the console.log.

Why can't you give it a proper ID that is unique? Please adapt, that we can use getElementByID.

Adapt for the other example.

Reply:
- Var to const updated
- Console.log removed
- type=module is currently breaking, wasn't changed due to it

> Why can't you give it a proper ID that is unique? Please adapt, that we can use getElementByID.
Due to storybook duplication of the stories, there are two ids that are not possible to be prevented. Bug ticket for it: https://github.com/solid-design-system/solid/issues/1566